### PR TITLE
[ZEPPELIN-1182] Zeppelin should have Startup and Shutdown message

### PR DESCRIPTION
**Pull request by rajarajan-g**
### What is this PR for?
PR is for logging configuration details of Zeppelin server.

### What type of PR is it?
Improvement

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1182

### How should this be tested?
Please check the log if information on Server host, server path, context path, zeppelin version, class path is available in that file.

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

Note:
    For configuration details such as  zeppelin version, server host, port & context path , these are available as part of zeppelin configuration. 

Logging information

```
 INFO [2016-09-02 14:35:25,218] ({main} ZeppelinConfiguration.java[create]:107) - Server Host: 0.0.0.0
 INFO [2016-09-02 14:35:25,218] ({main} ZeppelinConfiguration.java[create]:108) - Server Port: 8080
 INFO [2016-09-02 14:35:25,218] ({main} ZeppelinConfiguration.java[create]:109) - Context Path: /
 INFO [2016-09-02 14:35:25,224] ({main} ZeppelinConfiguration.java[create]:110) - Zeppelin Version: 0.7.0-SNAPSHOT
```

For Zeppelin class path and restart information,  I am logging it from Shell script to log file as in below paragraph, So the format of logging is different . I know this is not per standards, Please let me know if you have any suggestions
```
Zeppelin is restarted

ZEPPELIN_CLASSPATH: /home/rajarajang/Workspace/stsWorksapce/zeppelin/zeppelin-server/target/lib/*:/home/rajarajang/Workspace/stsWorksapce/zeppelin/zeppelin-zengine/target/lib/*:/home/rajarajang/Workspace/stsWorksapce/zeppelin/zeppelin-interpreter/target/lib/*:/home/rajarajang/Workspace/stsWorksapce/zeppelin/*::/home/rajarajang/Workspace/stsWorksapce/zeppelin/conf:/home/rajarajang/Workspace/stsWorksapce/zeppelin/zeppelin-interpreter/target/classes:/home/rajarajang/Workspace/stsWorksapce/zeppelin/zeppelin-zengine/target/classes:/home/rajarajang/Workspace/stsWorksapce/zeppelin/zeppelin-server/target/classes
```

### CODE REVIEW

**Comment by jongyoul** [comment]
Done or WIP?

**Comment by rajarajan-g** [comment]
I am logging the configuration details of Zeppelin . I am not sure if we have any more configuration details to be logged. Please check PR and let me know if any more configuration needs to be added.

**Comment by rajarajan-g** [comment]
@all: Please do review and let me know your opnion

**Comment by rajarajan-g** [comment]
Ping

File: bin/zeppelin-daemon.sh [hunk 1]
```diff
@@ -190,6 +190,7 @@ function start() {
   wait_zeppelin_is_up_for_ci
   sleep 2
   check_if_process_is_alive
+  echo "ZEPPELIN_CLASSPATH: ${ZEPPELIN_CLASSPATH}" >> "${ZEPPELIN_LOGFILE}"
```
Comment (Leemoonsoo) [inline bin/zeppelin-daemon.sh]: I think printing entire classpath i.e `$ZEPPELIN_CLASSPATH_OVERRIDES:$CLASSPATH` is more make sense. what do you think?

**Comment by rajarajan-g** [comment]
It makes more sense to have  message logged in ```${ZEPPELIN_OUTFILE}``` rather of ```${ZEPPELIN_LOGFILE}```.

File: bin/zeppelin-daemon.sh (hunk 1 above)
Comment (rajarajan-g) [inline bin/zeppelin-daemon.sh]: Sure.

But one quick question, Do we really set or use ```ZEPPELIN_CLASSPATH_OVERRIDES``` ? I don't see any assignment operation on that property. Please let me know if  i am missing something.

Comment (Leemoonsoo) [inline bin/zeppelin-daemon.sh]: Yes, classpath includes, `ZEPPELIN_CLASSPATH_OVERRIDES` https://github.com/apache/zeppelin/blob/master/bin/zeppelin-daemon.sh#L180
https://github.com/apache/zeppelin/blob/master/bin/interpreter.sh#L161

**Comment by Leemoonsoo** [comment]
CI failure is irrelevant. 
LGTM and merge if there're no further discussions.

**Pull request closed without merging**